modules/cnf-running-the-performance-creator-profile.adoc

```diff
@@ -78,55 +87,61 @@ Flags:
+$ podman run --entrypoint performance-profile-creator -v <path_to_must_gather>/must-gather:/must-gather:z registry.redhat.io/openshift4/ose-cluster-node-tuning-rhel9-operator:v4.16 --info log --must-gather-dir-path /must-gather
 ----
 +
-[NOTE]
-====
-This command uses the performance profile creator as a new entry point to `podman`. It maps the `must-gather` data for the host into the container image and invokes the required user-supplied profile arguments to produce the `my-performance-profile.yaml` file.
-
-The `-v` option can be the path to either of the following components:
-
-* The `must-gather` output directory
-* An existing directory containing the `must-gather` decompressed .tar file
-
-The `info` option requires a value which specifies the output format. Possible values are log and JSON. The JSON format is reserved for debugging.
-====
-
-. Run `podman`:
+* `--entrypoint performance-profile-creator` defines the performance profile creator as a new entry point to `podman`.
+* `-v <path_to_must_gather>` specifies the path to either of the following components:
+** The `must-gather` output directory.
+** An existing directory containing the `must-gather` decompressed .tar file.
+* `--info log` specifies a value for the output format.
++
+.Example output
+[source,terminal]
+----
+level=info msg="Cluster info:"
+level=info msg="MCP 'master' nodes:"
+level=info msg=---
+level=info msg="MCP 'worker' nodes:"
+level=info msg="Node: host.example.com (NUMA cells: 1, HT: true)"
+level=info msg="NUMA cell 0 : [0 1 2 3]"
+level=info msg="CPU(s): 4"
+level=info msg="Node: host1.example.com (NUMA cells: 1, HT: true)"
+level=info msg="NUMA cell 0 : [0 1 2 3]"
+level=info msg="CPU(s): 4"
+level=info msg=---
+level=info msg="MCP 'worker-cnf' nodes:"
+level=info msg="Node: host2.example.com (NUMA cells: 1, HT: true)"
+level=info msg="NUMA cell 0 : [0 1 2 3]"
+level=info msg="CPU(s): 4"
+level=info msg=---
+----
+
+. Create a performance profile by running the following command:
 +
 [source,terminal,subs="attributes+"]
 ----
-$ podman run --entrypoint performance-profile-creator -v /must-gather:/must-gather:z registry.redhat.io/openshift4/ose-cluster-node-tuning-operator:v{product-version} --mcp-name=worker-cnf --reserved-cpu-count=4 --rt-kernel=true --split-reserved-cpus-across-numa=false --must-gather-dir-path /must-gather --power-consumption-mode=ultra-low-latency --offlined-cpu-count=6 > my-performance-profile.yaml
+$ podman run --entrypoint performance-profile-creator -v /must-gather:/must-gather:z registry.redhat.io/openshift4/ose-cluster-node-tuning-rhel9-operator:v4.16 --mcp-name=worker-cnf --reserved-cpu-count=4 --rt-kernel=true --split-reserved-cpus-across-numa=false --must-gather-dir-path /must-gather --power-consumption-mode=ultra-low-latency --offlined-cpu-count=6 > my-performance-profile.yaml
```

Comment:
@MarSik - when I follow the procedure as doc'd in a Clusterbot metal instance I get the following error when I try to run this command:

`Error: failed to compute the MCP selector: can't find a unique label for 'worker-cnf' MCP`

Reply:
The worker-cnf MCP does not have a label that is unique among MCPs. You need to create one (anything works).

Reply:
Thanks, I figured this out eventually. I also added a another note reminding users that the PPC relies on the `must-gather` data. If you need to add labels, or make any changes that might impact the PP, then you need to re-create the `must-gather` data. This tripped me up quite a bit....